internal/servicehash/servicehash.go

```diff
@@ -0,0 +1,56 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package servicehash
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"sync"
+)
+
+const (
```

Comment:
You are using the grouping notation here (`(`) for just one constant, but elsewhere in the code you are using no grouping for multiple constants. Seems a bit counter intuitive.

NIT!